gui/src/2D/controls/inputPassword.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference path="../../../../dist/preview release/babylon.d.ts"/>
+
+module BABYLON.GUI {
+    /**
+     * Class used to create a password control
+     */
+    export class InputPassword extends InputText {
+        protected _beforeRenderText(text: string): string {
+            let txt = "";
+            for (let i = 0; i < text.length; i++) {
+                txt += "\u2022";
+            }

```

Comment:
nit: you could do "\u2022".repeat(text.length)

Reply:
That's what I did at first but the build complained about the repeat method not being available (because of an ES5 target I presume). Hence the loop.